configure.ac

```diff
@@ -1390,3 +1390,6 @@ m4_ifdef([project_ompi], [OMPI_CONFIG_FILES])
 m4_ifdef([project_oshmem], [OSHMEM_CONFIG_FILES])
 
 AC_OUTPUT
+
+OPAL_LOG_MSG([libnl : $opal_libnl_libs])
+OPAL_LOG_MSG([libnl3: $opal_libnl3_libs])
```

Comment:
I don't think my comment was clear; let me try again...

Perhaps in the section right before the AC_OUTPUT, where we show the various wrapper compiler flags, you can have something like this:
```sh
AC_MSG_CHECKING([for libraries that use libnl])
AC_MSG_RESULT([$opal_libnl_libs])
AC_MSG_CHECKING([for libraries that use libnl3])
AC_MSG_RESULT([$opal_libnl3_libs])
```

This would put it in both the stdout and in config.log.